### PR TITLE
knobs version updated

### PR DESCRIPTION
**Pull request by nicflores**
Knobs version updated to 4.0.30

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -137,5 +137,5 @@ object Dependencies {
     CommonDependencies.argonaut.monocle                         % Test,
     CommonDependencies.http4s.blazeClient                       % Test,
     CommonDependencies.refined.scalacheck                       % Test,
-    "io.verizon.knobs" %% "core"                % "3.12.27a"    % Test)
+    "io.verizon.knobs" % "core_2.11"  % "4.0.30-scalaz-7.2"     % Test)
```
Comment (jedesah) [inline project/Dependencies.scala]: Why the change to `%` over `%%`?

Comment (nicflores) [inline project/Dependencies.scala]: I tried staying sticking with our `%%` pattern but the only thing that worked was what is posted [here](http://search.maven.org/#artifactdetails%7Cio.verizon.knobs%7Ccore_2.11%7C4.0.30-scalaz-7.1%7Cjar). Using the original style:

```
"io.verizon.knobs" %% "core"  % "4.0.30-scalaz-7.2"         % Test
```

kept doubling up the core version

```
[trace] Stack trace suppressed: run last it/*:coursierResolution for the full output.
[error] (it/*:coursierResolution) coursier.ResolutionException: Encountered 1 error(s) in dependency resolution:
[error]   io.verizon.knobs:core_2.11_2.11:4.0.30-scalaz-7.2:
```

Let me know if I'm missing something.

Comment (drostron) [inline project/Dependencies.scala]: The one you posted in your comment is going to be what you want. I just gave it a whirl and seems ok. Maybe you had both `%` and `core_2.11` when you encountered the error.

Comment (nicflores) [inline project/Dependencies.scala]: Thanks @drostron and @jedesah for the heads up. What I posted above works so I went ahead and changed it.

**Comment by drostron** [comment]
@nicflores [This](https://github.com/quasar-analytics/quasar/blob/56ff1b6e7643a5ae5c87e68ac4f6c344b9527071/it/testing.conf.example#L1-L4) can now be removed.

File: it/testing.conf.example [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # This property is never used, but it's here so that Knobs doesn't fail to parse the file if all other
```
Comment (jedesah) [inline it/testing.conf.example]: The comments above should be removed as well
